docs/integrations/data-integrations/amazon-redshift.mdx

```diff
@@ -5,11 +5,11 @@ sidebarTitle: Amazon Redshift
 
 This is the implementation of the Redshift data handler for MindsDB.
 
-[Amazon Redshift](https://docs.aws.amazon.com/redshift/latest/mgmt/welcome.html) is a fully managed, petabyte-scale data warehouse service in the cloud. You can start with just a few hundred gigabytes of data and scale to a petabyte or more. This enables you to use your data to acquire new insights for your business and customers.
+[Amazon Redshift](https://docs.aws.amazon.com/redshift/latest/mgmt/welcome.html) is a fully managed, petabyte-scale data warehouse service in the cloud. You can start with just a few hundred gigabytes of data and scale to a petabyte or more enabling you to use your data to acquire new insights for your business and customers.
```

Comment:
There should be a comma here: `...to a petabyte or more, enabling you to use...` to make it correct.

Reply:
@martyna-mindsdb , thank you for catching that.  The missing comma has been added.